core/src/test/java/org/frankframework/jdbc/dbms/DbmsSupportTest.java

```diff
@@ -687,38 +687,6 @@ public void testIsClobType() throws SQLException {
 		}
 	}
 
-	@DatabaseTest
```

Comment:
Twee tests verwijderd omdat:
1. Ze worden alleen uitgevoerd als tabel `IBISTEMP` bestaat, maar die wordt niet gemaakt in de test changelogs, dus wordt de test altijd overgeslagen
2. Er zijn al andere tests die dezelfde check doen op de test-data tabel, dus het fixen van deze twee tests is niet nodig.